#filename_pcd: '/home/abdullah/catkin_ws/src/nbv_exploration/models/pcd/profile_cloud/profile_cloud_box_00_deg_tilt.pcd'
#filename_pcd: '/home/abdullah/catkin_ws/src/nbv_exploration/models/pcd/profile_cloud/profile_cloud_box_10_deg_tilt.pcd'
#filename_pcd: '/home/abdullah/catkin_ws/src/nbv_exploration/models/pcd/profile_cloud/profile_cloud_circular_00_deg_tilt.pcd'
#filename_pcd: '/home/abdullah/catkin_ws/src/nbv_exploration/models/pcd/profile_cloud/profile_cloud_circular_10_deg_tilt.pcd'
filename_pcd: '/home/abdullah/.ros/profile_cloud.pcd'

## Prefilter input
use_prefilter: false
prefilter_leaf_size: 0.05

## Keypoint settings
use_sift_points: true
sift_min_contrast: 0.0001
sift_min_scale: 0.1
sift_num_octaves: 3
sift_num_scales_per_octave: 4

#Percent of all points to sample, when SIFT is not used
pairing_subset_percent: 0.2

## Pairing settings
use_exhaustive_pairing: true
pairing_threshold: 10.0

## ICP (mirror image correction)
icp_with_normals: true
icp_max_euclid: 0.01
icp_max_transform: 0.001
icp_max_distance: 1

## Other settings
tree_K: 50
mean_shift_kernel_bandwidth: 10.0
mean_shift_num_points: 50 #-1 to use all points in clustering

#Radius to use when determining which points were newly added
subtraction_search_radius: 0.1
